index.js

```diff
@@ -137,15 +137,15 @@ function provider ({ Biome, blocks, blocksByStateId, toolMultipliers, shapes, ma
 
   function enchantmentLevel (enchantment, enchantments) {
     for (const e of enchantments) {
-      if (e.id.includes(enchantment)) {
+      if (typeof enchantment === 'string' ? e.id.includes(enchantment) : e.id === enchantment) {
         return e.lvl
       }
     }
     return -1
   }
 
   // http://minecraft.gamepedia.com/Breaking#Calculation
-  Block.prototype.digTime = function (heldItemType, creative, inWater, notOnGround, enchantments = [], effects = {}) {
+  Block.prototype.digTime = function (heldItemType, creative, inWater, notOnGround, enchantments = [], effects = {}, version) {
```

Comment:
You don't have to give the version, you can use `majorVersion` instead (defined here https://github.com/PrismarineJS/prismarine-block/blob/e88ba1ebd5d3f775b51322b4f03e4309a9024622/index.js#L17). That should remove the breaking change.